arabic/salamoAlikom.md

```diff
@@ -9,3 +9,5 @@
 [اريد ان اتعلم المزيد](learn/learn.md)
 
 [اريد ان احلم](dream/dream.md)
+
+و إن كنا نريد أن نسرد قصة قصيرة فهي قصة أمتنا ووطننا الذي يعاني التمزيق و الضياع, نريد أن تكون تلك الخطوات التى نخطوها و ان كانت بسيطة إنطلاقة لتحقيق نهضة علمية لأمتنا.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.